src/Features/CSharp/Portable/AddImport/CSharpAddImportFeatureService.cs

```diff
@@ -288,7 +290,7 @@ protected override string GetDescription(IReadOnlyList<string> nameParts)
 
         protected override (string description, bool hasExistingImport) GetDescription(
             Document document,
-            OptionSet options,
+            CodeGenerationPreferences preferences,
```

Comment:
containing type is generic.  could make it parameterized on this type so the subclass doesn't need to cast.

Reply:
In this case adding the extra generic param is not a small change and it saves just one cast, so I'll keep it as is.